src/NuGetForUnity.Tests/Assets/NuGet.config

```diff
@@ -12,6 +12,7 @@
     <add key="repositoryPath" value="./Packages" />
     <add key="DefaultPushSource" value="http://www.nuget.org/api/v2/" />
     <add key="verbose" value="true" />
+    <add key="PackagesConfigPath" value="Assets" />
```

Comment:
When we store a path inside a file the "expected" behavior is that the path is relative to the file that contains the configuration. For `repositoryPath` it is the same. So the value `.` means the `package.config` file is placed in the same folder as the `NuGet.config` file.

Reply:
Fixed the behaviour now with this.